### ISSUE TITLE
Get Client constructors public

### ISSUE THREAD

**Issue by Rdornier** [GReD-Clermont/simple-omero-client#34]
Hello @ppouchin,

We are trying to create an instance of the Client object without asking the user his credentials. 
In our project, the user logged-in and the gateway is kept active while his credentials are lost. At some point, we need to create a Client object from the simple-omero-client API. For convienance, we do not want to ask the user to log-in twice. 

I saw that Client object has 2 constructors but the interested one is private. For now, we use java reflection methods to get access to the private constructor (you can see how [here](https://github.com/BIOP/qupath-extension-biop-omero/blob/33e90af3a2ed5b5e69fac5d9beef1db1b087cabd/src/main/java/qupath/ext/biop/servers/omero/raw/OmeroRawTools.java#L511)).

In order to make a cleaner code, I was thinking to put this constructor public.
 ```
Client(Gateway gateway, SecurityContext ctx, ExperimenterWrapper user) {
        super(gateway, ctx, user);
    }
```

What do you think about that ? Is it possible ?

Thanks,
Rémy.

ping @lacan, @romainGuiet, @NicoKiaru

**Comment by ppouchin** [comment]
Ok. Why did I make that private? I wonder...
I added this when I split the Client in "GatewayWrapper" and "Client" classes.

I don't think there's a reason why it should remain private. I'll add it to the 5.9.2 to-do. 
Alternatively, it is possible to [connect to an existing session](https://github.com/GReD-Clermont/simple-omero-client/blob/main/src/main/java/fr/igred/omero/GatewayWrapper.java#L178) with the session ID, but I think there are limitations (group switching for example).

**Comment by Rdornier** [comment]
Ok, great !
Thank you very much !

I just tried the connection with the session ID and it also works.

**Comment by ppouchin** [comment]
Should be solved in 5.9.2.
